microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md

```diff
@@ -0,0 +1,469 @@
+
+| Feature | Microsoft Edge version | Platform |  
+|:--- |:--- |:--- |  
+| [URL Protocol Handling](#url-protocol-handling) | 91 or later | Windows |  
+| [Window Controls Overlay for Installed Desktop Web Apps](#window-controls-overlay-for-installed-desktop-web-apps) | 91 or later | Windows |  
+| [Run on OS Login](#run-on-os-login) | 88 or later | All |  
+| [Shortcuts](#shortcuts) | 87 or later | All |  
+| [File Handling](#file-handling) | 83 or later | All Desktop|  
+
+## Features Coming Soon  
+
+This table includes features that are work in progress that may come out in a future release for testing.  
+
+| Feature | Platform |  
+|:--- |:--- |   
+| [URL Link Handling](#url-link-handling) | Windows |  
+| [Window Controls Overlay for Installed Desktop Web Apps](#window-controls-overlay-for-installed-desktop-web-apps) |  macOS, Linux |  
+| Improvements to App Management| All |  
+| Support for Desktop Share Target| Windows 10 |  
+
+## URL Protocol Handling  
+
+URLs can be used to define more than just links to webpages and web content using the HTTP or FTP protocol - they can be used to describe links to anything you can codify into a schema. For example, the `mailto://` protocol is used to describe an email link and the operating system or browser can decide which webpage or app should handle that protocol. You can learn more about existing browser based support for [web based protocol handling here](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler/Web-based_protocol_handlers). 
+
+This feature allows you to register a PWA with the host operating system via the web app manifest, declaring that it can handle a specific URL protocol. After registering a PWA as a protocol handler, when a user chooses a hyperlink with a specific scheme like `mailto://`, `ms-word://` or `web+music://` from a browser or a native app, the registered PWA would be activated by the operating system and receive the URL.
+
+This feature requires you to update the web app manifest to include a `protocol_handlers` array, within the array you need to specify two fields:
+
+*   `protocol`: Protocol to be handled e.g. `mailto`, `web+jngl`  
+*   `url`: HTTPS URL within the app scope that will handle the protocol. The `%s` token will be replaced by the URL starting with the protcol handlers scheme.  
+
+### Example Manifest
+
+In this example, a web app manifest declares that the app should be registered to handle the protocols `web+jngl` and `web+jnglstore`.
+
+```json
+{
+  "name": "Jungle",
+  "description": "A plant encyclopedia",
+  "protocol_handlers": [
+    {
+      "protocol": "web+jngl",
+      "url": "/lookup?type=%s"
+    },
+    {
+      "protocol": "web+jnglstore",
+      "url": "/shop?for=%s"
+    }
+  ],
+  "icons": [
+    {
+      "src": "images/icons-44.png",
+      "type": "image/png",
+      "sizes": "44x44"
+    },
+    {
+      "src": "images/icons-144.png",
```

Comment:
Too much space in this section wasted with icons (which is irrelevant to the feature). Should we remove 2 of these entries and leave just an icon?

Reply:
ok can do